.github/workflows/check-milestone.yml

```diff
@@ -0,0 +1,80 @@
+name: Milestone Reminder
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - "master"
+
+jobs:
+  milestone-reminder:
+    if: github.event.pull_request.merged == 'true'
+    name: Remind to set milestone
+    runs-on: ubuntu-22.04
+    timeout-minutes: 5
+    permissions:
+      pull-requests: write
+      issues: read
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: | # js
+            const owner = context.repo.owner;
+            const repo = context.repo.repo;
+
+            const pullNumber = '${{ github.event.pull_request.number }}';
+
+            // the github API doesn't expose linked issues, so we have to parse the body ourselves
+            function getLinkedIssueId(pull) {
+              // https://regexr.com/7mjnp
+              const match = pull.data.body.match(/(\/issues\/|#)(\d+)/);
+              return match && match?.[2];
```

Comment:
Ideally we should:
- handle a case where PR closes multiple issues
- match [GitHub's keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) because PRs will often mention other related issues that are not necessarily milestone-related
- skip links to other repositories

Reply:
All of those are great ideas. I realize there's lots of unhandled edge cases here, but I'd rather ship the minimum viable version of this first before building something more complex. I think this will cover 95% of our use cases.